### PR TITLE
[SRS] Quotes functional requirements

### PR DESCRIPTION
**Pull request by arkinmodi**
Close https://github.com/arkinmodi/project-sayyara/issues/72

### CODE REVIEW

**Review by joyxiao99 — APPROVED** [review]

File: docs/SRS/SRS.tex [hunk 1]
```diff
@@ -334,6 +335,223 @@ \subsubsection{Authentication}
...
+\begin{enumerate}[resume*=business_events]
+	\item The user wants view past quotes
+	      \begin{enumerate}[VP\arabic*.]
+		      \item Viewpoint: Vehicle Owner
+		            \begin{enumerate}
+			            \item The system shall list all quotes
+		            \end{enumerate}
+		      \item Viewpoint: Auto Repair Shop Owner
+		            \begin{enumerate}
+			            \item The system shall allow the user to enter the quote ID, customer phone number, and customer name
```
Comment (LeonSo7) [inline docs/SRS/SRS.tex:348]: should we auto generate a quote id?

File: docs/SRS/SRS.tex [hunk 2]
```diff
@@ -334,6 +335,223 @@ \subsubsection{Authentication}
...
+		            \begin{enumerate}
+			            \item[] N/A
+		            \end{enumerate}
+		      \item Viewpoint: Auto Repair Shop Employee
+		            \begin{enumerate}
+			            \item[] N/A
+		            \end{enumerate}
+	      \end{enumerate}
+
+	\item A quote request has been created
```
Comment (LeonSo7) [inline docs/SRS/SRS.tex:409]: a business event should be trigger outside of the system right?

File: docs/SRS/SRS.tex [hunk 3]
```diff
@@ -334,6 +335,223 @@ \subsubsection{Authentication}
...
+		            \begin{enumerate}
+			            \item[] N/A
+		            \end{enumerate}
+		      \item Viewpoint: Auto Repair Shop Employee
+		            \begin{enumerate}
+			            \item[] N/A
+		            \end{enumerate}
+	      \end{enumerate}
+
+	\item The user wants to send an existing quote request to another shop
```
Comment (LeonSo7) [inline docs/SRS/SRS.tex:459]: maybe say copy a quote request?

File: docs/SRS/SRS.tex [hunk 4]
```diff
@@ -334,6 +335,223 @@ \subsubsection{Authentication}
...
+		            \begin{enumerate}
+			            \item The system shall list active quotes
+			            \item The system shall allow the user to update a quote
+		            \end{enumerate}
+		      \item Viewpoint: Auto Repair Shop Owner
+		            \begin{enumerate}
+			            \item[] N/A
+		            \end{enumerate}
+		      \item Viewpoint: Auto Repair Shop Employee
+		            \begin{enumerate}
```
Comment (LeonSo7) [inline docs/SRS/SRS.tex:454]: should employees be notified of a change, so they are aware?

File: docs/SRS/SRS.tex [hunk 5]
```diff
@@ -334,6 +335,223 @@ \subsubsection{Authentication}
...
+			            \item The system shall allow the user to enter the estimated price, a list of services, the estimated
+			                  time, a list of required parts, and discounts
+			            \item The system shall automatically apply local taxes
+			            \item The system shall send the quote to the customer
+			            \item The system shall send a notification to the customer
+		            \end{enumerate}
+	      \end{enumerate}
+
+	\item A quote request has been responded to
```
Comment (LeonSo7) [inline docs/SRS/SRS.tex:503]: BE should be an external trigger

**Review by LeonSo7 — APPROVED** [review]

**Pull request merged**